README.md

```diff
@@ -721,9 +721,9 @@ Int. J. Computer Vision (IJCV), 2021.
 - <a name="Wang20arxiv"></a>Wang, Z., Ng, Y., Scheerlinck, C., Mahony., R.,  
 *[An Asynchronous Kalman Filter for Hybrid Event Cameras](https://arxiv.org/abs/2012.05590)*,  
 arXiv, 2020.
-- <a name="Wang20arxivb"></a>Wang, Z., Ng, Y., van Goor, P., Mahony., R.,  
+- <a name="Wang19acra"></a>Wang, Z., Ng, Y., van Goor, P., Mahony., R.,  
 *[Event Camera Calibration of Per-pixel Biased Contrast Threshold](https://arxiv.org/pdf/2012.09378)*,  
-arXiv, 2020.
+Australasian Conference on Robotics and Automation (ACRA) 2019.
```

Comment:
Thanks! Please abbreviate the conference, for consistency with the name of publication venue used by Stoffregen in 2017:
Australasian Conf. Robotics and Automation (ACRA)

Reply:
Thank you for the suggestion. It has been fixed.